util/otbnsim/setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(name="otbnsim",
       packages=find_packages(),
-      install_requires=["riscv-model>=0.6-rc3", "lark-parser"],
+      install_requires=["riscv-model>=0.6", "lark-parser"],
```

Comment:
Since these are all being merged together, it probably makes more sense to bump the dependency on the first commit.